### DISCUSSION TITLE
Open Source Summit (North America)

### DISCUSSION THREAD

**Discussion by dwgray** [cdmoro/bootstrap-vue-3#1841] [General]
Is anyone else attending the [Open Source Summit - North America](https://events.linuxfoundation.org/open-source-summit-north-america/) in Seattle next week? If so, any interest in a BSVN meet-up?

**Comment by VividLemon** [comment]
Although it sounds like fun, it's a bit too short notice for me to attend. Although, I'd like to attend events like this. It's something I've never done

**Comment by dwgray** [comment]
I wasn't expecting anyone to pick up and fly to Seattle on short notice; I was just checking if anyone else was going for other reasons. I'm local and am doing some [volunteer work](https://mentorsintech.com/), which gave me the opportunity to sign up and go fairly last minute.
